sys-fs/ncdu/ncdu-2.1.2.ebuild

```diff
@@ -14,7 +14,7 @@ KEYWORDS="~amd64"
 DEPEND="sys-libs/ncurses:=[unicode(+)]"
```

Comment:
(This needs a revbump really to allow people to change to -bin, just git mv for it)